modules/services/espanso.nix

```diff
@@ -122,8 +122,17 @@ in {
       Unit = { Description = "Espanso: cross platform text expander in Rust"; };
       Service = {
         Type = "exec";
```

Comment:
The upstream source doesn't specify `Type` so presumably you could remove this line?

Reply:
Done. I've also added `RestartSec` as per source.